src/app/page.tsx

```diff
@@ -1,5 +1,51 @@
-"use client";
+import HomeWrapper from "@/components/HomeWrapper";
+import FaqCard from "@/components/FaqCard";
+import { MOCK_FAQ } from "@/jsondata/faqMock";
+import Image from "next/image";
+import separator from "../../public/decorating/shapes/separator.svg";
 
 export default function Home() {
-  return <>Helloooooo</>;
+  return (
+    <HomeWrapper>
+      <div className="mt-12 flex flex-col gap-5 text-red">
+        <div className="type-title-large font-medium">
+          <p>Intania First Date 2025</p>
+          <p>19 กรกฎาคม 2568</p>
+        </div>
+        <Image src={separator} className="mx-auto" priority alt="Separator" />
+        <div className="type-title-large font-medium">
+          <p>และค่ายวิษณุกรรมบุตรครั้งที่ 23</p>
+          <p>22–25 กรกฎาคม 2568</p>
+        </div>
+      </div>
+
+      <div className="pb-30 z-30 w-full">
+        <div className="type-title-large text-center font-bold mt-17 mb-5 text-red">
+          <p>คำถามที่พบบ่อย</p>
+        </div>
+        <div className="flex flex-col items-center gap-6">
+          {MOCK_FAQ.map((item, index) => (
+            <div key={index} className="w-full  ">
+              {item.title.length > 0 && (
+                <div className="mb-4 bg-[url('/decorating/shapes/flag.svg')] bg-no-repeat w-42 h-10">
+                  <p className="type-title-medium font-bold text-left pl-7 py-2 text-white">
+                    {item.title}
+                  </p>
+                </div>
```

Comment:
This might be better as a component since we’ll reuse it in the Manage Users page. 

Also you accidentally clipped the right edge.

![](https://github.com/user-attachments/assets/fd168e94-2363-4abf-9f11-da17f316170c)